includes/Upgrader/Upgrades/V_2_0_2.php

```
@@ -1,118 +1,118 @@
+            'edit_post',
+            'edit_docs',
+            'publish_docs',
+            'edit_others_docs',
+            'read_private_docs',
+            'edit_private_docs',
+            'edit_published_docs'
+        );
+
+        // Push documentation handling access to users.
+        foreach ( $capabilities as $capability ) {
+            foreach ( $roles as $role_key => $role ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$role'. (undefined)

(UnusedLocalVariable)